spec/facility_spec.rb

```diff
@@ -23,4 +32,81 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register vehicle' do
+    it 'has registered vehicles' do
+      expect(@facility_1.registered_vehicles).to eq([])
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
+    end
+
+    it 'only registers a vehicle if the facility offers that service' do
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.register_vehicle(@bolt)).to eq(nil)
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.collected_fees).to eq(0)
```

Comment:
Great job testing that you can not register and none of these attributes are effected if that first condition isn't met.